src/modules/function/invocation_utils.rs

```diff
@@ -28,11 +28,14 @@ fn ordinal_number(index: usize) -> String {
 fn run_function_with_args(meta: &mut ParserMetadata, mut fun: FunctionDecl, args: &[Type], tok: Option<Token>) -> Result<(Type, usize), Failure> {
     // Check if there are the correct amount of arguments
     if fun.arg_names.len() != args.len() {
+        let max_args = fun.arg_names.len();
+        let min_args = fun.arg_names.len() - fun.arg_optionals.len();
+        let opt_argument = if max_args > min_args {&format!("({} optional)",max_args)} else {""};
```

Comment:
I just realized that the space should be added here instead

```suggestion
        let opt_argument = if max_args > min_args {&format!(" ({} optional)",max_args)} else {""};
```